### PR TITLE
fix-up-pkg-document-note-format .. -> .

### PR DESCRIPTION
**Pull request by tanjunchen**
### Ⅰ. Describe what this PR did
fix-up the /pkg document note .. -> . 

### Ⅱ. Does this pull request fix one issue?

None

### Ⅲ. Why don't you add test cases (unit test/integration test)? (你真的觉得不需要加测试吗？)

None

### Ⅳ. Describe how to verify it
None

### Ⅴ. Special notes for reviews

None

### CODE REVIEW

File: pkg/fileutils/fileutils.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
-// BufferSize define the buffer size when reading and writing file
+// BufferSize define the buffer size when reading and writing file.
```
Comment (Starnop) [inline pkg/fileutils/fileutils.go]: s/define/defines

File: pkg/netutils/netutils.go [hunk 2]
```diff
@@ -282,7 +282,7 @@ func isExist(mmap map[string]bool, key string) bool {
 //
 // The 0 will be returned when both minRate and defaultMinRate both are <=0.
 func CalculateTimeout(fileLength int64, minRate int, defaultMinRate int, reservedTime time.Duration) time.Duration {
-	// ensure the minRate to avoid trigger panic when minRate equals zero
+	// ensure the minRate to avoid trigger panic when minRate equals zero.
```
Comment (Starnop) [inline pkg/netutils/netutils.go]: I think there is no need to add a period when the comment is not a complete sentence.

File: pkg/fileutils/fileutils.go [hunk 3]
```diff
@@ -190,7 +190,7 @@ func IsRegularFile(name string) bool {
 	return f.Mode().IsRegular()
 }
 
-// Md5Sum generate md5 for a given file
+// Md5Sum generate md5 for a given file.
```
Comment (Starnop) [inline pkg/fileutils/fileutils.go]: Use third person singular. THX.

File: pkg/netutils/netutils.go [hunk 4]
```diff
@@ -41,7 +41,7 @@ const (
 
 var defaultRateLimit = "20M"
 
-// NetLimit parse speed of interface that it has prefix of eth
+// NetLimit parse speed of interface that it has prefix of eth.
```
Comment (Starnop) [inline pkg/netutils/netutils.go]: Use third person singular. THX.

Comment (tanjunchen) [reply pkg/netutils/netutils.go]: OK thanks

**Comment by Starnop** [comment]
LGTM. @tanjunchen Thanks for your good job.

**Pull request merged**